zwave_js_server/model/node.py

```diff
@@ -337,6 +344,7 @@ def handle_value_removed(self, event: Event) -> None:
 
     def handle_value_notification(self, event: Event) -> None:
         """Process a node value notification event."""
+        event.data["value"] = ValueNotification.from_event(event)
```

Comment:
Should we store this under a different key since it's a value notification and not a value?

Reply:
Not sure. It is basically the same as a regular value except it's stateless.
Perhaps store it in "notification" key is better ?

Reply:
Yes, I think we should use a different key since it's a different instance.

Reply:
done!